src/components/Auth/SignUp/SignUp.react.js

```diff
@@ -280,24 +295,22 @@ export default class SignUp extends Component {
             });
     }
     render() {
-
-        const serverURL = <TextField name="serverUrl"
-                            onChange={this.handleChange}
+        const customUrlStyle= {
+            width:'202px',
+            textAlign:'left',
+            margin:'-35px 0 0px 5px',
+        }
+        const serverURL = <TextField
```

Comment:
@isuruAb Add a onClick function and handle the this.state.checked state there to achieve the effect.